server/odc-core/src/main/java/com/oceanbase/odc/core/sql/util/FullLinkTraceUtil.java

```diff
@@ -67,8 +70,8 @@ private static SqlExecTime parseSpanList(List<TraceSpan> spanList) {
                     return v;
                 });
             }
-            if (traceId == null && parseLogTraceId(span.getTags()) != null) {
-                traceId = parseLogTraceId(span.getTags());
+            if (traceId == null && findLogTraceId(span.getTags()) != null) {
```

Comment:
Why execute `findLogTraceId(span.getTags())` twice?

Reply:
I don't know 😂, and I just refactor these codes to another way that can be executed on ob 4.2.1. @LuckyPickleZZ may answer this question

Reply:
okay, I find another way to get traceId that execute `findLogTraceId(span.getTags())` only once